### DISCUSSION TITLE
Expected output for the sample script

### DISCUSSION THREAD

**Discussion by boorah** [savonet/liquidsoap#2907] [Q&A] [answered]
Hello everyone,
I wanted to try out `liquidsoap` in my local machine and was going through the `Liquidsoap Book`. On running the following piece of code - 

```
#!/usr/bin/env liquidsoap

output(sine())
```

I am getting the following output -
![image](https://user-images.githubusercontent.com/39365143/219716151-1691fbf4-751b-4653-b627-a55fc8d6e355.png)

Can you please tell me what I might be missing here?

I am using Fedora 37

Thanks

**Comment by vitoyucepi** [comment, marked as answer]
Hi @boorah,
It seems there's no *oss* in your system.
Have you tried other output methods
- [`output.pulseaudio`](https://www.liquidsoap.info/doc-2.1.3/reference.html#output.pulseaudio)
- [`output.alsa`](https://www.liquidsoap.info/doc-2.1.3/reference.html#output.alsa)
- [`output.jack`](https://www.liquidsoap.info/doc-2.1.3/reference.html#output.jack)
- [`output.ao`](https://www.liquidsoap.info/doc-2.1.3/reference.html#output.ao)
- [`output.portaudio`](https://www.liquidsoap.info/doc-2.1.3/reference.html#output.portaudio)

If I were you, I would prefer [`output.harbor`](https://www.liquidsoap.info/doc-2.1.3/reference.html#output.harbor).

**Reply by boorah** [reply]
Thanks a lot! `output.harbor` works :)
